todo_app/data/trello_items.py

```diff
@@ -1,15 +1,26 @@
 import requests
 import os
 from datetime import datetime
+from pymongo import MongoClient
 
 from .session_items import *
 
 
-# globals
-baseUrl = "https://api.trello.com/1/"
-headers = {"Accept": "application/json"}
+#Get Connection sTrign for Mongo
 
+def get_connection_string():
+    return os.environ.get("CONNECTION_STRING")
 
+def get_collection(databaseNem, collectionName):
+    return databaseNem[collectionName]
+
+#Globals
+client= MongoClient(get_connection_string())
+db = client['ToDo-Database']
+cardsCollection = get_collection(db, 'todo-cards')
+boardsCollection = get_collection(db, 'todo-boards')
```

Comment:
Having these as globals might be the reason that your tests feel slightly odd (e.g. end-to-end uses a fake DB) - it can cause fragility because they get set as soon as this file is imported, which means there's not much point wrapping the environment variables above in functions. Instead we probably want to wrap them somehow.

For example we could solve this by creating a wrapper class for this file that looks something like below:
```python
class MongoAccess():
    def __init__(self):
        conn_str = os.environ.get("CONNECTION_STRING")
        client= MongoClient(conn_str)
        db = client['ToDo-Database']
        self.cardsCollection = db['todo-cards']
        self.boardsCollection = db['todo-boards']

    # Add a new document to the DBcard
    def add_card(self, listId, cardName, desc, due, board, addedCollection = None):
        collection = self.cardsCollection
        ...
```

(Separately, I'd advise pulling the DB name out into an environment variable - it feels like that would be nice for ensuring that we can use the same CosmosDB account for different environments (in a real project I might advise against this anyway, but for us being cheap with a single one it's definitely helpful!)